artiq/coredevice/phaser.py

```diff
@@ -1,6 +1,8 @@
+from numpy import int32, int64
```

Comment:
remove int64

Reply:
It is used here https://github.com/pathfinder49/artiq/blob/e74f1d20f6af03ddd3dab706948c4218d7e183f4/artiq/coredevice/phaser.py#L360
TBH, I'm not sure how the current code compiles given its not imported.

Reply:
Yes. Some globals are available in ARTIQ-Python beyond the standard Python globals.

Reply:
Keep them like this then.